From a93cc7232431cdcc3ed4c41bf7e7afba981ae278 Mon Sep 17 00:00:00 2001
From: Louis Lenders <xerox.xerox2000x@gmail.com>
Date: Sun, 27 Apr 2025 22:29:33 +0200
Subject: [PATCH] shell32: Change type of KNOWNFOLDERID
 FOLDERID_UserProgramFiles from CSIDL_Type_Disallowed to CSIDL_Type_User.

Wine-Bug: https://bugs.winehq.org/show_bug.cgi?id=58169
---
 dlls/shell32/shellpath.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/shell32/shellpath.c b/dlls/shell32/shellpath.c
index 1a52717651e..dad5949fdce 100644
--- a/dlls/shell32/shellpath.c
+++ b/dlls/shell32/shellpath.c
@@ -2039,7 +2039,7 @@ static const CSIDL_DATA CSIDL_Data[] =
     },
     { /* 0x6e */
         .id         = &FOLDERID_UserProgramFiles,
-        .type       = CSIDL_Type_Disallowed, /* FIXME */
+        .type       = CSIDL_Type_User,
         .category   = KF_CATEGORY_PERUSER,
         .name       = L"UserProgramFiles",
         .parent     = &FOLDERID_LocalAppData,
-- 
GitLab

